doc/antora/modules/concepts/pages/components/nas.adoc

```diff
@@ -1,58 +1,23 @@
 = Network Access Server
 
-A Network Access Server (NAS) is a system that provides access to
-a network. In some cases, it's also known as a Terminal Server or Remote
-Access Server (RAS).
+A Network Access Server (NAS) is a system that grants or denies access to
+a protected resource. It is sometimes referred to as a Terminal Server (TS) or a Remote Access Server (RAS). The NAS acts as a gateway that controls access to resources such as networks, printers, or the internet.
 
-The NAS is meant to act as a gateway to guard access to a
-protected resource. This can be anything from a telephone network,
-to printers, to the Internet.
+When a network user or device needs access to a resource, it first sends a request to the NAS. The NAS doesn't store information about who is allowed or what credentials are valid. Instead, it forwards the credentials to a RADIUS server capable of AAA services for validation. Based on the RADIUS server's response, the NAS either grants or denies access to the requested resource. For example, if the credentials are valid, access is granted.
 
-The client connects to the NAS. The NAS then connects to another
-resource asking whether the client's supplied credentials are
-valid. Based on that answer the NAS then allows or disallows
-access to the protected resource.
+== Examples
 
-The NAS contains no information about what clients can connect or
-what credentials are valid. All the NAS does is send the
-credentials the client supplied to a resource which does know how
-to process the credentials.
+There are many NAS implementations that handle requests for access such as:
 
-== Examples
+* A user opens their browser to get online. The NAS controls access to the Internet and checks to see if the user is authorized. If not, the NAS prompts the user to enter their credentials such as a username and password. The NAS forwards these credentials to a RADIUS server for validation. If the RADIUS server confirms the credentials are valid, the NAS allows the user to access the Internet. This process ensures that only authorized users can use network resources.
```

Comment:
The user connects to the NAS somehow (ADSL, Wi-Fi, etc.).  We shouldn't mention "browser" here, as it isn't involved with gaining network access.